configs/stash-cache/config.d/40-stash-cache-plugin.cfg

```
@@ -80,6 +80,20 @@ else
    set eSPACEHIGHWM=0.95
 fi
 
+if defined ?~WQ_BLOCKS_PER_LOOP
+   set eWQBLOCKS=$WQ_BLOCKS_PER_LOOP
+else
```

```suggestion
if defined ?~XC_WQ_BLOCKS
   set eWQBLOCKS=$XC_WQ_BLOCKS
else
```

To follow formatting of other vars